src/core/MOM_dynamics_split_RK2.F90

```diff
@@ -326,6 +333,7 @@ subroutine step_MOM_dyn_split_RK2(u_inst, v_inst, h, tv, visc, Time_local, dt, f
   type(thickness_diffuse_CS),        intent(inout) :: thickness_diffuse_CSp !< Pointer to a structure containing
                                                                    !! interface height diffusivities
   type(porous_barrier_type),         intent(in)    :: pbv          !< porous barrier fractional cell metrics
+  type(stochastic_CS),               intent(inout) :: STOCH        !< Stochastic control structure
```

Comment:
The only place in `step_MOM_dyn_split_RK2()` where `stoch` is used is as an optional argument to `horizontal_viscosity()`, so why is it not an optional argument to `step_MOM_dyn_split_RK2()`?  Please consider making the new `stoch` argument optional, given that it is not commonly used in most existing MOM6 applications.

Reply:
It looks like we missed this comment. I've now opened an issue in the NCAR/MOM6 repository (https://github.com/NCAR/MOM6/issues/350). Given the timeline and all the approvals, I'll merge this PR as is, and we'll surely address this comment in our next PR to main.

Reply:
Well, somehow, clicking the "Resolve conversation" button doesn't work for me. @Hallberg-NOAA do you mind clicking the "Resolve conversation" button to see if it works and if you are okay with it. Or, we can actually address this (by Monday), so please let us know.

Reply:
I did this a few weeks ago. It is resolved in this PR. See https://github.com/NCAR/MOM6/blob/982b5776a51bed07f9d811092c4b8b8fbb65960e/src/core/MOM_dynamics_split_RK2.F90#L336